hw/opentitan/ot_spi_device.c

```diff
@@ -1977,11 +2085,158 @@ static void ot_spi_device_chr_recv_flash(OtSPIDeviceState *s,
     }
 }
 
+static bool ot_spi_device_tpm_is_hw_register(uint32_t addr)
+{
+    static const uint32_t TPM_HW_ADDR[] = {
+        0x000u, /* 000     Access_x*/
+        0x008u, /* 00B:008 Interrupt Enable*/
+        0x00Cu, /* 00C     Interrupt Vector*/
+        0x010u, /* 013:010 Interrupt Status*/
+        0x014u, /* 017:014 Interface Capability*/
+        0x018u, /* 01B:018 Status_x*/
+        0x028u, /* 02B     Hash Start*/
```

Comment:
```suggestion
        0x028u, /* 028     Hash Start*/
```
(sorry to be a pain!)